### PR TITLE
chore: update steamcmd example script

### PR DESCRIPTION
**Pull request by gOOvER**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changelog:

- updated installimage
- updated package install to match bullseye
- move force_install_dir to beginning

### CODE REVIEW

File: scripts/steamcmd_installer.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # steamcmd Base Installation Script
 #
 # Server Files: /mnt/server
-# Image to install with is 'debian:buster-slim'
+# Image to install with is 'ghcr.io/parkervcp/games:source'
```
Comment (Software-Noob) [inline scripts/steamcmd_installer.sh:5]: This is not an install image

File: scripts/steamcmd_installer.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # steamcmd Base Installation Script
 #
 # Server Files: /mnt/server
-# Image to install with is 'ghcr.io/parkervcp/games:source'
+# Image to install with is 'ghcr.io/parkervcp/yolks:debian'
```
Comment (Software-Noob) [inline scripts/steamcmd_installer.sh:5]: The image to install would be `ghcr.io/parkervcp/installers:debian`. Yolks:debian is a runtime image
```suggestion
# Image to install with is 'ghcr.io/parkervcp/installers:debian'
```

**Review by Software-Noob — APPROVED** [review]
LGTM, thank you for contributing :+1:

File: scripts/steamcmd_installer.sh (hunk 1 above)
Comment (Egor-oop) [reply scripts/steamcmd_installer.sh]: fhfhfhfhfhfhfhhfhfhhffhhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhffhhfhf

**Pull request merged**